src/main/java/Expense.java

```diff
@@ -1,17 +1,91 @@
 package src.main.java;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public class Expense {
     private int expenseId;
     private String description;
     private double amount;
     private User paidBy;
+    private Group group;
+    private List<User> benefactors;
+    private double individualShare;
 
-    public Expense(int expenseId, String description, double amount, User paidBy) {
+    public Expense(int expenseId, String description, double amount, User paidBy, Group group) {
         this.expenseId = expenseId;
         this.description = description;
         this.amount = amount;
         this.paidBy = paidBy;
+        this.group = group;
+        this.benefactors = new ArrayList<>();
+    }
```

Comment:
`individualShare` wasnt initialized here.

Reply:
Thanks for pointing it our. I had missed this and I have resolved it and pushed again. Kindly check it out.